mon-pix/app/templates/competence-details.hbs

```diff
@@ -0,0 +1,54 @@
+{{navbar-header class="navbar-header--white"}}
+<div class="competence-details">
+  <div class="background-banner"></div>
+
+  {{#if (and model.scorecard model.competence)}}
```

Comment:
Pas besoin de ce if je pense.

Reply:
Je trouve ça plus propre de ne rien afficher lorsque le model est absent plutôt que d'afficher une page à moitié chargé (sans le contenu, où tout est petit et entassé)

Reply:
Si je dis pas de bêtises Ember ne commence pas le rendering avant d'avoir chargé le model.

Reply:
Effectivement, le template n'est rendu que lorsque le `model hook` est complètement chargé. En revanche si on navigue back and forth, Ember l'ayant déjà en cache va tout de suite la rendre et aller la `background_reload` (sauf si configuré autrement). Et enfin, si le template mentionne des objets qu'Ember n'a pas et qu'on a pas demandé dans le model, il ira les chercher en temps réel et c'est là où on peut avoir des pb d'UI qui blink. Il me semble.

Reply:
Ember commence le rendering avant que le model soit entièrement chargé j'ai vérifié. Surtout si le model est une promesse.